IBF-Typhoon-model/.gitignore

```diff
@@ -10,3 +10,5 @@ lib/climada_python-main
 home/
 otherfiles/
 temp/
+model_trigger.py
+credentials
```

Comment:
You should consolidate all of these to a single .gitignore at the top level of the repository